internal/server/data/data_test.go

```diff
@@ -76,66 +75,8 @@ func TestSnowflakeIDSerialization(t *testing.T) {
 	})
 }
 
-func TestDatabaseSelectors(t *testing.T) {
```

Comment:
Removed because this test is just a simulation of the selector behaviour, and we should be removing selector functions soon. Porting this test to the new interface would be a lot more work than other tests, and is not worth it.